src/ui/util/saved-passwords.js

```diff
@@ -37,16 +37,23 @@ function deleteCookie(name: string) {
 
 export const setSavedPassword = (value?: string, saveToDisk: boolean) => {
   return new Promise<*>(resolve => {
+    // @if TARGET='app'
     ipcRenderer.once('set-password-response', (event, success) => {
       resolve(success);
     });
+    // @endif
 
     const password = value === undefined || value === null ? '' : value;
     sessionPassword = password;
 
     if (saveToDisk) {
       if (password) {
+        // @if TARGET='app'
         ipcRenderer.send('set-password', password);
+        // @endif
+        // @if TARGET='web'
+        setCookie('saved-password', password, 14);
```

Comment:
I should have caught this in an earlier review, but it was only reading this that triggered me. A lot of this file speaks generically of "saved password" or "saved passwords", but in reality this file only allows for the setting of a single password and a very specific one at that: your wallet password.

Reply:
Also auth_token